src/exercises/exerciseTypes/NextNavigation.js

```diff
@@ -48,8 +44,12 @@ export default function NextNavigation({
   const [learningCycle, setLearningCycle] = useState(null);
   const [showCelebrationModal, setShowCelebrationModal] = useState(false);
   const [isDeleted, setIsDeleted] = useState(false);
-  const [autoPronounceState, autoPronounceString, toggleAutoPronounceValue] =
-    useBookmarkAutoPronounce();
+  const [
+    isPronounceBookmark,
+    currentPronouncingState,
```

Comment:
`currentPronouncingState` does not seem to be used in the file. do we still need it here? 

i guess the code could be written like: 

```js
    [isPronounceBookmark, ,
    autoPronounceString,
    toggleAutoPronounceValue] = ...
```

although that could be less readable?